packages/apps/src/app.process.ts

```diff
@@ -158,7 +158,7 @@ export async function $process<TPlugin extends IPlugin>(
     await context.stream.close();
 
     if (!res || !isInvokeResponse(res)) {
-      res = { status: 200, body: res ?? {} };
+      res = { status: 200, body: res };
```

Comment:
`body: res` - res can still be null in this case right?

Reply:
yeah if the application decides to send back null, it can be null. But if the application decides to send back undefined, we don't set it to null